docs/INSTALL.md

```diff
@@ -133,44 +133,73 @@ This command may be useful if you decided to use a separate database server.
 
 ## Starting Plume
```

Comment:
We should have fixed the info on how to compile Plume with sqlite in #226, why DM i only thinking of this now?